src/main/java/com/team2357/lib/subsystems/LimelightSubsystem.java

```diff
@@ -19,10 +32,11 @@ public class LimelightSubsystem extends ClosedLoopSubsystem {
 
   public static class Configuration {
 
-    public int m_humanPipelineIndex = 0;
+    public int m_humanPipelineIndex = 2;
 
     public int m_targetingPipelineIndex = 0;
 
+    public int m_aprilTagPipelineIndex = 1;
```

Comment:
This value should probably be defined in Constants rather than here.

Reply:
Same as above, this is in a config class, so we can always just override it. Do we want a specific constants file in robotlib and get rid of the configuration class?